src/gnatss/loaders.py

```diff
@@ -260,7 +260,7 @@ def load_gps_solutions(
 
 
 def load_deletions(
-    file_path: str, config: Configuration, time_scale="tt"
+    file_path: Optional[str], config: Configuration, time_scale="tt"
```

Comment:
```suggestion
    file_path: str, config: Configuration, time_scale="tt"
```
This shouldn't be optional. Adding the term `Optional` means that it can either by an actual `str` value or `None`. By default you're passing in empty string here `''`.